README.md

```diff
@@ -1351,6 +1345,10 @@ drive report-issue -title "Can't open my file" -body "Drive trips out every time
 cat bugReport.txt | drive issue -piped -title "push: dump on pushing from this directory"
 ```
 
+### Automating
```

Comment:
You already have this section in https://github.com/odeke-em/drive/pull/872/files#diff-04c6e90faac2675aa89e2176d2eec7d8R162.